src/prefect/orion/database/migrations/versions/postgresql/2022_11_24_143620_f7587d6c5776_add_worker_tables.py

```diff
@@ -0,0 +1,239 @@
+"""Add worker tables
+
+Revision ID: f7587d6c5776
+Revises: 5d526270ddb4
+Create Date: 2022-11-24 14:36:20.350834
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+import prefect
+
+# revision identifiers, used by Alembic.
+revision = "f7587d6c5776"
+down_revision = "5e4f924ff96c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "worker_pool",
+        sa.Column(
+            "id",
+            prefect.orion.utilities.database.UUID(),
+            server_default=sa.text("(GEN_RANDOM_UUID())"),
+            nullable=False,
+        ),
+        sa.Column(
+            "created",
+            prefect.orion.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated",
+            prefect.orion.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column("description", sa.String(), nullable=True),
+        sa.Column("type", sa.String(), nullable=True),
+        sa.Column("is_paused", sa.Boolean(), server_default="0", nullable=False),
+        sa.Column("concurrency_limit", sa.Integer(), nullable=True),
+        sa.Column(
+            "default_queue_id", prefect.orion.utilities.database.UUID(), nullable=True
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_worker_pool")),
+        sa.UniqueConstraint("name", name=op.f("uq_worker_pool__name")),
+    )
+    op.create_index(
+        op.f("ix_worker_pool__updated"), "worker_pool", ["updated"], unique=False
+    )
+    op.create_index(op.f("ix_worker_pool__type"), "worker_pool", ["type"], unique=False)
+
+    op.create_table(
+        "worker",
+        sa.Column(
+            "id",
+            prefect.orion.utilities.database.UUID(),
+            server_default=sa.text("(GEN_RANDOM_UUID())"),
+            nullable=False,
+        ),
+        sa.Column(
+            "created",
+            prefect.orion.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated",
+            prefect.orion.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("CURRENT_TIMESTAMP"),
+            nullable=False,
+        ),
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column(
+            "last_heartbeat_time",
```

Comment:
The worker table feels a bit heavy to me, especially if we're going to be updating it with every poll, but I understand why it's helpful to have